components/questions/CommentForm.tsx

```
@@ -0,0 +1,191 @@
+interface CommentFormProps {
+  questionId: string
+  mode: 'create' | 'edit'
+  initialContent?: string
+  commentId?: string
+  onSubmit: (content: string, commentId?: string) => Promise<void>
+  onCancel?: () => void
+  isSubmitting?: boolean
+}
+
+export function CommentForm({
+  questionId,
```

ESLint

[error] 'questionId' is defined but never used.

(@typescript-eslint/no-unused-vars)